src/NetAnalyzers/CSharp/Microsoft.CodeQuality.Analyzers/Maintainability/CSharpUseCrossPlatformIntrinsics.Fixer.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System;
+using System.Composition;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeQuality.Analyzers.Maintainability;
+
+namespace Microsoft.CodeQuality.CSharp.Analyzers.Maintainability
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp), Shared]
+    public sealed class CSharpUseCrossPlatformIntrinsicsFixer : UseCrossPlatformIntrinsicsFixer
+    {
+        protected override SyntaxNode CreateExclusiveOrExpression(SyntaxNode left, SyntaxNode right)
```

Comment:
I logged https://github.com/dotnet/roslyn/issues/79314 for these missing APIs.

VB is notably lacking `UnsignedRightShift` but there's otherwise nothing really "blocking" it from working there. We just don't have any known users there today and it would be a significant number of almost duplicate tests that need to be written, so I opted to not add that support right now.

Reply:
I agree with keeping VB out of scope while not necessarily going out of our way to make it hard to add VB support in the future should demand arise (albeit unlikely).